docs/troubleshooting/harvester.md

```diff
@@ -4,7 +4,53 @@ sidebar_label: Harvester
 title: ""
 ---
 
-## Generate a support bundle
+## Fail to Deploy a Multi-node Cluster due to Incorrect HTTP Proxy Setting
+
+### ISO Installation Without a Harvester Configuration File
+
+#### Configure HTTP Proxy During Harvester Installation
+
+In some environments, you configure [http-proxy](../airgap/#configure-an-http-proxy-during-installation) of [OS Environment](../install/harvester-configuration.md#osenvironment) during Harvester installation.
+
+#### Configure HTTP Proxy After First Node is Ready
+
+After the first node is installed successfully, you login into the `Harvester GUI` to configure [http-proxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) of [Harvester System Settings](../install/harvester-configuration.md#system_settings).
```

Comment:
```suggestion
After the first node is installed successfully, you log in to the `Harvester GUI` to configure [http-proxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) of [Harvester System Settings](../install/harvester-configuration.md#system_settings).
```

Reply:
`login` means input `user credential like username and password`, it is a defined word in IT. please keep it unchanged.

Reply:
The definition you listed is correct, but login is a noun or adjective. In the context here, it's being used as a verb.

Reply:
`login` is the noun or adjective form while the verb form is `log in`. In any case, if you want to keep it as `login` the next word should be `to` instead of `into`.